### PR TITLE
add script to renew all albs at once

### PR DESCRIPTION
**Pull request by bengerman13**
## Changes proposed in this pull request:
- add script to renew all albs at the same time
- add docs about the new scripts
- update alb_certs.py to bail instead of fail

## security considerations
None

### CODE REVIEW

File: certbot/README.md [hunk 1]
```diff
@@ -1,8 +1,32 @@
 # Less-manual manual cert rotation tools
 
-## Usage
+## all-in-one
 
-Both scripts expect you have set the environment variable CERTBOT_BUCKET_NAME to be the name
+*Note* This uses bash-isms not available on the default OS X bash (3.x).
```
Comment (ccostino) [inline certbot/README.md:5]: Missing words:

```suggestion
*Note* This uses bash-isms, which is not available on the default OS X bash (3.x). 
```

File: certbot/README.md [hunk 2]
```diff
@@ -1,8 +1,32 @@
 # Less-manual manual cert rotation tools
 
-## Usage
+## all-in-one
 
-Both scripts expect you have set the environment variable CERTBOT_BUCKET_NAME to be the name
+*Note* This uses bash-isms not available on the default OS X bash (3.x). 
+In order to run either script, make sure you've installed a newer version of
+bash with homebrew
```
Comment (ccostino) [inline certbot/README.md:7]: Homebrew instructions for installing BASH:

```suggestion
bash with homebrew: `brew install bash`
```

**Review by ccostino — APPROVED** [review]
Thanks, @bengerman13!  Two minor suggestions if they're helpful, otherwise :shipit:

File: certbot/README.md (hunk 1 above)
Comment (bengerman13) [reply certbot/README.md:5]: this suggestion doesn't work, but it does tell me that my original wording isn't great, so I'll tweak it

Comment (ccostino) [reply certbot/README.md]: No worries!

**Pull request merged**